documentation/airport-format.md

```diff
@@ -1,179 +1,168 @@
+            // United Kingdom, etc., be sure to flag all the transition fixes.
     }
   },
-  "stars": {  // contains all STARS available at this airport
+  "stars": {                                // STARs are STandard ARrival paths for aircraft to follow.
     "PYE1" : {
-      "icao": "PYE1",               // (req) ICAO identifier for SID (this is NOT the full name, always 2-6 characters)
-      "name": "Point Reyes One" ,   // (req) Name of SID as it would be said aloud (it is used by speech synthesis to pronounce "OFFSH9")
-      "suffix": {"1L":"", "1R":"", "28L":"", "28R":""},   // (optional) defines suffixes to STAR name based on runway (eg '7W' for 'MIKOV 7W').
-                                                          // Common for European-style STARs. If not needed (like in USA), leave this part out.
-      "entryPoints": {    // (optional) Defines entryPoints for a given SID. Common for FAA-style (USA) SIDs. If not needed (like in Europe), leave this part out.
-          "ENI": ["ENI"], // defines the "OFFSH9.SNS" transition as being a single fix, "SNS". Is often a list instead.
-          "MXW": ["MXW"]  // Note that this connects to the end of previous sections, so an example route: SEPDY->ZUPAX->EUGEN->SHOEY->BSR
+      "icao": "PYE1",                       // ICAO identifier of the STAR. (NOT the full name, always 2-6 characters)
+      "name": "Point Reyes One",            // Name of STAR as it would be said aloud. (used by speech synthesis)
+      "suffix": {"1L":"", "1R":"", "28L":"", "28R":""},   // (optional) Suffixes for STAR names based on a runway. (ex: '7W' for 'MIKOV 7W')
+                                                          // Suffixes are common for European-style STARs. If not needed (like in USA), leave this part out.
+      "entryPoints": {                      // (optional) Defines entryPoints for a given SID. Common for FAA-style (USA) SIDs.
+          "ENI": ["ENI"],                   // Defines the "PYE1.ENI" transition as being a single fix, "ENI". It is often a list instead.
+          "MXW": ["MXW"]                    // Note: This connects previous sections. (ENTRYPOINTS fixes, then BODY fixes, then RWY fixes) (ex: ENI->PYE->STINS->HADLY->SEPDY->ZUPAX)
       },
-      "body": ["PYE", ["STINS", "A230|S250"], "HADLY"], // (optional) This is where you store the waypoints when all segments are along the same path
-      "rwy": {  // (optional) For runway-transitions (eg "descending via HAWKZ4, landing north")
-          "1L" : [["SEPDY", "A19+"], "ZUPAX"],  // List fixes here that are specific to a particular runway configuration.
-          "1R" : [["SEPDY", "A19+"], "ZUPAX"],  // If any runways are listed, all must be listed.
-          "28L": [["SENZY", "A25+"], "ZUPAX"],
-          "28R": [["SENZY", "A25+"], "ZUPAX"]
+      "body": ["PYE", ["STINS", "A230|S250"], "HADLY"], // (optional) Used only if long series of fixes are used. Only used while all segments follow the same path.
```

Comment:
Add in the brackets that this is NOT optional if the entryPoint(s) have only got one waypoint.